components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1161,17 +1193,14 @@ private Condition createConditionForSingleAttributeFilter(String domainName, Nod
      * @return
      * @throws CharonException
      */
-    private List<Object> getMultiAttributeFilteredUsers(Node node, Map<String, Boolean> requiredAttributes,
-                                                        int offset, int limit, String sortBy, String sortOrder,
-                                                        String domainName, List<Object> filteredUsers)
+    private List<Object> getMultiAttributeFilteredUsers(Node node, Map<String, Boolean> requiredAttributes, int offset,
+            int limit, String sortBy, String sortOrder, String domainName, List<Object> filteredUsers)
             throws CharonException {
 
         String[] userNames;
-
         // Handle pagination.
         if (limit > 0) {
-            userNames = getFilteredUsersFromMultiAttributeFiltering(node, offset, limit, sortBy,
-                    sortOrder, domainName);
+            userNames = getFilteredUsersFromMultiAttributeFiltering(node, offset, limit, sortBy, sortOrder, domainName);
```

Comment:
Exceeds line length

Reply:
The formatter does not recognize it has an exceeded. It seems here that it exceeds the limit. But in IDEA it isn't.